coralproject/cay
# Gallery design tweaks

## What does this PR do?

This PR addresses the design updates in #406. What it doesn't do is make extra buttons work. For example, you still can't publish a gallery to Elkhorn yet, that's going to come in a separate PR so this one doesn't get too unwieldy.

Missing things:
- There is no concept yet of Created By in the schema, so we'll have to talk to @jde about it.
- I have not yet added the white gradient on the boxes.
- clicking the identity checkboxes doesn't do anything yet, and we'l have to figure out exactly what we want to happen there.
- up/down arrows don't do anything yet.
- I talked to @egrdina an there won't be an ability to clone answers (the copy button)

## How do I test this PR?

- check out the branch and make sure stuff looks right and that i didn't miss anything else.

@coralproject/frontend
